sdk/iot/Azure.Iot.Hub.Service/src/API Design.md

```diff
@@ -64,8 +64,43 @@ APIs for using IotHub v2 jobs
 
 <details><summary><b>Messages</b></summary>
 Feedback messages, sending cloud to device messages (missing from current swagger), and purging cloud to device message queue
-```csharp
 
+```csharp
+public class Messages
```

Comment:
Will this only be about CloudToDevice messages?

Reply:
yes; should I rename the client to reflect that?

Reply:
Something to consider